### PR TITLE
Update README of `package:process`

### PR DESCRIPTION
**Pull request by bartekpacia**
This PR fixes https://github.com/flutter/flutter/issues/136221.

BTW it's a great package! I use it extensively in [patrol_cli] and love it.

[patrol_cli]: https://pub.dev/packages/patrol_cli

### CODE REVIEW

**Review by stuartmorgan — CHANGES_REQUESTED** [review]
This requires a version change, per our contributing docs. (I would suggest not deleting the checklist from the PR description in the future, as it covers required steps like that one.)

File: packages/process/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 5.0.2
 
 * Updates minimum supported SDK version to Flutter 3.10/Dart 3.0.
+* Remove mention of the removed record/replay feature from README.
```
[the remark this answers is not in the document]
Comment (bartekpacia) [reply packages/process/CHANGELOG.md:4]: Sorry again, I missed it.

[the remark this answers is not in the document]
Comment (stuartmorgan) [reply packages/process/CHANGELOG.md:5]: This was copied instead of moved.

**Review by stuartmorgan — CHANGES_REQUESTED** [review]

File: packages/process/CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bartekpacia) [reply packages/process/CHANGELOG.md:5]: 🤦🏻

[the remark this answers is not in the document]
Comment (stuartmorgan) [reply packages/process/CHANGELOG.md:4]: You deleted the new, corrected one (which addressed my original comment) instead of the old one.

**Comment by stuartmorgan** [comment]
@bartekpacia Are you still planning on updating this per the comments above?

**Comment by bartekpacia** [comment]
@stuartmorgan yes, I pushed the changes.

**Pull request merged**